### PR TITLE
Bullet Updates and scrap ammo

### PR DESCRIPTION
**Pull request by shazbot194**
## About The Pull Request
Behold, a major update to sprites. This majorly updates icons, stacking of ammo, adds scrap ammo, as well as scrap ammo kits. All values are subject to change as they are my best guess for balance.

icons_3
 Above are all the full-size sprites, however, this is now how they look in-game, instead, they are shrunk to be more fitting to the original sprite size. 

icons_2
This is how they look in-game. You might notice they are clumped together. This is because all shells can now be stacked like shotgun shells or ATMR casings, Pistols can have 15, magnums 6, and rifles 10. This also makes gunfight cleanup easier as spent casings autostack on tiles. 

In all the ammo, the steel/grey casing ammo is scrap ammo. This ammo spawns in maintenance and can be made with ammo kits. They do 90% of the damage normal lethal does and there are limited options for ammo, but it is a lot easier to acquire for vagabonds. 

icons_1
The Scrap Ammo Kit lets you make bullets based on your cognition score. It uses spawn tables for all but shotgun shells so you can get ammo boxes or handfuls of shells.

## Why It's Good For The Game
This is a major update to icons, makes loose ammo more convenient, as well as adds in all the systems needed for scrap ammo and adds scrap ammo.

## Changelog
:cl:
add: Added scrap ammo and scrap ammo kits.
add: Added stacking for normal bullets
imageadd: Added unique icons for all ammo
/:cl:

### CODE REVIEW

File: code/modules/projectiles/ammunition/bullets.dm [hunk 1]
```diff
@@ -2,173 +2,375 @@
...
+
 /obj/item/ammo_casing/antim/prespawned
 	amount = 5
 
+/obj/item/ammo_casing/antim/s
+	name = "shell casing"
+	desc = "An old .60 Anti-Material shell."
+	icon_state = "atmr_s"
+	spent_icon = "atmr_s-spent"
+	projectile_type = /obj/item/projectile/bullet/antim/owo
```
Comment (Firefox13) [inline code/modules/projectiles/ammunition/bullets.dm:266]: I'm sure it should use a different projectile.

Comment (shazbot194) [reply code/modules/projectiles/ammunition/bullets.dm:266]: Oops, must have made a mistake when I was tired. I will fix it when I get home tonight.

File: code/modules/projectiles/projectile/bullettypes.dm [hunk 2]
```diff
@@ -182,6 +197,19 @@ There are important things regarding this file:
 	penetrating = 5
 	hitscan = TRUE //so the PTR isn't useless as a sniper weapon
 
+/obj/item/projectile/bullet/antim/s
+	damage_types = list(BRUTE = 63)
+
+/obj/item/projectile/bullet/antim/owo
```
Comment (SpadesSlick) [inline code/modules/projectiles/projectile/bullettypes.dm:203]: No.

File: code/modules/projectiles/projectile/bullettypes.dm [hunk 3]
```diff
@@ -201,6 +232,16 @@ There are important things regarding this file:
 	embed = FALSE
 	sharp = FALSE
 
+/obj/item/projectile/bullet/shotgun/beanbag/s
+	damage_types = list(BRUTE = 9)
+	agony = 55
+
+/obj/item/projectile/bullet/shotgun/beanbag/bd
```
Comment (SpadesSlick) [inline code/modules/projectiles/projectile/bullettypes.dm:239]: No.

**Review by SpadesSlick — DISMISSED** [review]

File: code/modules/projectiles/projectile/bullettypes.dm (hunk 2 above)
Comment (shazbot194) [reply code/modules/projectiles/projectile/bullettypes.dm:203]: This has been approved by Reere 2.0, aka Firefox on the condition it's admin only.

File: code/modules/projectiles/projectile/bullettypes.dm (hunk 3 above)
Comment (shazbot194) [reply code/modules/projectiles/projectile/bullettypes.dm:242]: This has been approved by Reere 2.0, aka Firefox on the condition it's admin only.

**Comment by SpadesSlick** [comment]
Admin only things tend to find it's way into IC situations due bad code practices of developers who will slip them in by another loot rework, or just by plain admin shitspawn. This happens every year on Eris.

Admin items who have no actual value, and exist only for meme purposes would be fine on itself, if only it won't be a degenerative content that goes against our guidelines.

> Suggestive content is to be evaluated for it's impact on the codebase on a case by case basis, by reason and in accordance with rules stated below.

Bad dragon shotgun ammo brings nothing of value and enhance suicidal tendencies among people who will eventually find it. Not to mention that existence of such items can create a reddit tier drama, with their code being posted outside of "admin-only" context, in order to tarnish a reputation of our project among SS13 community.

Not to mention that there is no comments or clear indication that it's admin only item, so someone might actually ""

All of it applies to object that never supposed to be found in game, and by that, does not enhance game in any way even if it will stay this way, and only adds useless bloat.

> This has been approved by Reere 2.0, aka Firefox on the condition it's admin only.

If he desires to overrun my decision on this, he can discard my changes request.

**Comment by shazbot194** [comment]
After discussing it with Firefox, it has been removed.

**Review by mattroks101 — APPROVED** [review]
Code looks good. Unless @SpadesSlick finds any other memes I missed.

File: code/modules/projectiles/ammunition/ammo_kits.dm [hunk 4]
```diff
@@ -0,0 +1,233 @@
+
+//// .35 ////
+/obj/item/ammo_kit
+	name = "Scrap Ammo Kit"
+	desc = "A somewhat jank looking crafting kit. It has a can of signel use tools,cheap pliers, and a box of bullet making materials."
```
Comment (Anonmare) [inline code/modules/projectiles/ammunition/ammo_kits.dm:5]: *single use tools, cheap pliers and a box of bullets

Comment (Firefox13) [reply code/modules/projectiles/ammunition/ammo_kits.dm:5]:
```suggestion
	desc = "A somewhat jank looking crafting kit. It has a can of single-use tools, cheap pliers and a box of bullet making materials."
```

**Review by Anonmare — APPROVED** [review]

**Review by TheShown911 — APPROVED** [review]

**Pull request merged**